src/index.ts

```diff
@@ -436,11 +451,20 @@ class QueryPipeline extends EventEmitter {
           }
         }
 
+        let promotionPinned = false;
+        if (activePins.length > 0) {
+          const val = valueFromProto(values["_id"]) as string;
+          if (activePins.indexOf(val) !== -1) {
+            promotionPinned = true;
+          }
+        }
+
         return {
           indexScore,
           score,
           values: processProtoValues(values),
           token: t,
+          promotionPinned: promotionPinned,
```

Comment:
`promotionPinned,`

This shorthand just takes a variable name and makes a key with the same name.